<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reisduur-Rechner</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="Label_WindDir">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>100</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>Windrichtung:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Input_WindDir">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>120</width>
      <height>35</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Deg">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>16</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>°</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_TWS">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>150</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>Windgeschwindigkeit:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Input_TWS">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>120</width>
      <height>35</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Knots">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>50</y>
      <width>21</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>kn</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Box_Start">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>125</y>
      <width>141</width>
      <height>35</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="Box_Finish">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>125</y>
      <width>141</width>
      <height>35</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Result">
    <property name="geometry">
     <rect>
      <x>355</x>
      <y>210</y>
      <width>50</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>125</y>
      <width>51</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>Von</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>125</y>
      <width>31</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>nach</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>210</y>
      <width>50</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Dauer:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>210</y>
      <width>63</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Stunden</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Update">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>30</y>
      <width>94</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QWidget" name="downWidget" native="true">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>280</y>
      <width>590</width>
      <height>280</height>
     </rect>
    </property>
    <widget class="QLabel" name="Label_Laps">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>331</width>
       <height>201</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>201</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Gesamtdauer:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_SumClear">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>30</y>
       <width>94</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>210</y>
       <width>63</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>ges: </string>
     </property>
    </widget>
    <widget class="QLabel" name="Label_TotalTime">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>210</y>
       <width>63</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="Button_Plus">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>240</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Alter">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>125</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>→←</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
